lib/rubocop/cop/mixin/comments_help.rb

```diff
@@ -64,8 +64,10 @@ def buffer
       # the line at which the parent node ends.
       # rubocop:disable Metrics/AbcSize
```

Comment:
The following will be required for my suggestions:

```suggestion
      # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```